# Titanic competition with TensorFlow Decision Forests


# Imports dependencies

In [1]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
import tensorflow_decision_forests as tfdf

print(f"Found TF-DF {tfdf.__version__}")

Found TF-DF 1.8.1


# Load dataset

In [2]:
train_df = pd.read_csv("train.csv")
serving_df = pd.read_csv("test.csv")

train_df.head(10)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

# Prepare dataset

We will apply the following transformations on the dataset.

1. Tokenize the names. For example, "Braund, Mr. Owen Harris" will become ["Braund", "Mr.", "Owen", "Harris"].
2. Extract any prefix in the ticket. For example ticket "STON/O2. 3101282" will become "STON/O2." and 3101282.

In [3]:
def preprocess(df):
    df = df.copy()
    
    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])
    
    def ticket_number(x):
        return x.split(" ")[-1]
        
    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])
    
    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)                     
    return df
    
preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

list of the input features of the model

In [4]:
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")
#input_features.remove("Ticket_number")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


# Convert Pandas dataset to TensorFlow Dataset

In [5]:
def tokenize_names(features, labels=None):
    """Divite the names into tokens. TF-DF can consume text tokens natively."""
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

# Train model with default parameters

### Train model

train a GradientBoostedTreesModel model with the default parameters.

In [6]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    random_seed=1234,
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-10-30 03:29:50.7618 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:50.7633 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:50.7633 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-10-30 03:29:51.9814 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp2mod95vz/model/ with prefix ef5526be31c945ba
[INFO 24-10-30 03:29:51.9859 +07 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-10-30 03:29:51.9859 +07 kernel.cc:1061] Use fast generic engine


Accuracy: 0.804347813129425 Loss:0.8922085165977478


# Train model with improved default parameters

In [7]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    
    #num_trees=2000,
    
    # Only for GBT.
    # A bit slower, but great to understand the model.
    # compute_permutation_variable_importance=True,
    
    # Change the default hyper-parameters
    # hyperparameter_template="benchmark_rank1@v1",
    
    #num_trees=1000,
    #tuner=tuner
    
    min_examples=1,
    categorical_algorithm="RANDOM",
    #max_depth=4,
    shrinkage=0.05,
    #num_candidate_attributes_ratio=0.2,
    split_axis="SPARSE_OBLIQUE",
    sparse_oblique_normalization="MIN_MAX",
    sparse_oblique_num_projections_exponent=2.0,
    num_trees=2000,
    #validation_ratio=0.0,
    random_seed=1234,
    
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-10-30 03:29:52.3219 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:52.3219 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:52.3219 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


Accuracy: 0.77173912525177 Loss:1.0136744976043701


[INFO 24-10-30 03:29:52.5935 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp6z7od6yc/model/ with prefix 772332675f8e4c2e
[INFO 24-10-30 03:29:52.5976 +07 decision_forest.cc:660] Model loaded with 54 root(s), 2824 node(s), and 10 input feature(s).
[INFO 24-10-30 03:29:52.5976 +07 kernel.cc:1061] Use fast generic engine


variable importance that the model figured out

In [8]:
model.summary()

Model: "gradient_boosted_trees_model_1"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
Total params: 1 (1.00 Byte)
Trainable params: 0 (0.00 Byte)
Non-trainable params: 1 (1.00 Byte)
_________________________________________________________________
Type: "GRADIENT_BOOSTED_TREES"
Task: CLASSIFICATION
Label: "__LABEL"

Input Features (11):
	Age
	Cabin
	Embarked
	Fare
	Name
	Parch
	Pclass
	Sex
	SibSp
	Ticket_item
	Ticket_number

No weights

Variable Importance: INV_MEAN_MIN_DEPTH:
    1.           "Sex"  0.431313 ################
    2.           "Age"  0.373986 ############
    3.          "Fare"  0.250774 ####
    4.          "Name"  0.225272 ###
    5.   "Ticket_item"  0.182111 
    6.      "Embarked"  0.181387 
    7. "Ticket_number"  0.180897 
    8.        "Pclass"  0.178279 
    9.         "Parch"  0.175344 
   10.         "SibSp"  0.172167 

Variable Importance: NUM_AS_ROOT:
    1.  "Sex" 39.00000

# Make predictions

In [9]:
def prediction(model, threshold=0.5):
    proba_survive = model.predict(serving_ds, verbose=0)[:,0]
    return pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (proba_survive >= threshold).astype(int)
    })

def make_submission(kaggle_predictions):
    path="submission.csv"
    kaggle_predictions.to_csv(path, index=False)
    print(f"Submission exported to {path}")
    
kaggle_predictions = prediction(model)
make_submission(kaggle_predictions)
# !head /kaggle/working/submission.csv




Submission exported to submission.csv


# Making an ensemble


In [10]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    # Possible models: GradientBoostedTreesModel or RandomForestModel
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0, # Very few logs
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True, # Only use the features in "features"

        #min_examples=1,
        #categorical_algorithm="RANDOM",
        ##max_depth=4,
        #shrinkage=0.05,
        ##num_candidate_attributes_ratio=0.2,
        #split_axis="SPARSE_OBLIQUE",
        #sparse_oblique_normalization="MIN_MAX",
        #sparse_oblique_num_projections_exponent=2.0,
        #num_trees=2000,
        ##validation_ratio=0.0,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)
    
    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

make_submission(kaggle_predictions)

[WARNING 24-10-30 03:29:52.7380 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:52.7380 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:52.7380 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:0


[INFO 24-10-30 03:29:52.9069 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpg021uowx/model/ with prefix 3b9d25946499476b
[INFO 24-10-30 03:29:52.9083 +07 kernel.cc:1061] Use fast generic engine


[WARNING 24-10-30 03:29:53.0235 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:53.0235 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:53.0235 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:1


[INFO 24-10-30 03:29:53.2230 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpqaffdvj9/model/ with prefix 5efa8b3161c64810


i:2


[INFO 24-10-30 03:29:53.2261 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:53.3400 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:53.3400 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:53.3400 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:3


[INFO 24-10-30 03:29:53.5419 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpm3af17d4/model/ with prefix 4e4ca2ea455c43df
[INFO 24-10-30 03:29:53.5451 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:53.6593 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:53.6593 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:53.6593 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


[INFO 24-10-30 03:29:53.8363 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpaksruzis/model/ with prefix f9002e9583e94a21
[INFO 24-10-30 03:29:53.8386 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:53.9520 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:53.9520 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:53.9520 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:4


[INFO 24-10-30 03:29:54.0995 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpykjrcm5l/model/ with prefix ef76f754d88a41cd
[INFO 24-10-30 03:29:54.1008 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:54.2132 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:54.2132 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:54.2132 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:5


[INFO 24-10-30 03:29:54.3709 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp3pxykgfk/model/ with prefix 4f53db9318284e49
[INFO 24-10-30 03:29:54.3725 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:54.4851 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:54.4852 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:54.4852 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:6


[INFO 24-10-30 03:29:54.6978 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmptml3jfdh/model/ with prefix b085f99a7edd4a24
[INFO 24-10-30 03:29:54.7016 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:54.8273 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:54.8273 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:54.8273 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:7


[INFO 24-10-30 03:29:54.9972 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpgpll687x/model/ with prefix 7458e66e863044f9
[INFO 24-10-30 03:29:54.9991 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:55.1145 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:55.1145 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:55.1145 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:8


[INFO 24-10-30 03:29:55.3262 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpg74knchu/model/ with prefix c6fe3aef8572433b
[INFO 24-10-30 03:29:55.3305 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:55.4421 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:55.4421 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:55.4421 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:9


[INFO 24-10-30 03:29:55.6648 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp5d4ht9yh/model/ with prefix 20ca90f6118b413b
[INFO 24-10-30 03:29:55.6689 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:55.7821 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:55.7822 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:55.7822 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:10


[INFO 24-10-30 03:29:55.9499 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp3dmwaia4/model/ with prefix 04be516a4ceb4809
[INFO 24-10-30 03:29:55.9521 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:56.0658 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:56.0658 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:56.0658 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:11


[INFO 24-10-30 03:29:56.2924 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpk7gspnaz/model/ with prefix 7d781f65018b4488
[INFO 24-10-30 03:29:56.2967 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:56.4099 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:56.4100 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:56.4100 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:12


[INFO 24-10-30 03:29:56.5931 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp6iziihx4/model/ with prefix dfa56cf18fc243aa
[INFO 24-10-30 03:29:56.5958 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:56.7089 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:56.7090 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:56.7090 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:13


[INFO 24-10-30 03:29:57.1407 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp4bqvaz2w/model/ with prefix 56b7b76d38e740fd
[INFO 24-10-30 03:29:57.1489 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:57.2856 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:57.2856 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:57.2856 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:14


[INFO 24-10-30 03:29:57.4613 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmputp9w6oj/model/ with prefix ffac6aa6155d4867
[INFO 24-10-30 03:29:57.4636 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:57.5799 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:57.5799 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:57.5799 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:15


[INFO 24-10-30 03:29:57.7377 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpgehizmme/model/ with prefix 5c3fc356ae544b03
[INFO 24-10-30 03:29:57.7392 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:57.8551 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:57.8551 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:57.8551 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:16


[INFO 24-10-30 03:29:58.0534 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp0r_fybdt/model/ with prefix dcd7b00beb404d68
[INFO 24-10-30 03:29:58.0565 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:58.1726 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:58.1727 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:58.1727 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:17


[INFO 24-10-30 03:29:58.3355 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp_b51sp6w/model/ with prefix 42dfa00973854661
[INFO 24-10-30 03:29:58.3373 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:58.4530 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:58.4530 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:58.4530 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:18


[INFO 24-10-30 03:29:58.6639 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpwelwvil3/model/ with prefix 32e9db9f52464406
[INFO 24-10-30 03:29:58.6680 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:58.7822 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:58.7822 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:58.7822 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:19


[INFO 24-10-30 03:29:58.9769 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpd3k1e22b/model/ with prefix 8c55c86841e14f08
[INFO 24-10-30 03:29:58.9801 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:59.0940 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:59.0940 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:59.0940 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:20


[INFO 24-10-30 03:29:59.2553 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpfyna7jvr/model/ with prefix 1e5151b0cfd24759
[INFO 24-10-30 03:29:59.2571 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:59.3709 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:59.3709 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:59.3709 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:21


[INFO 24-10-30 03:29:59.5533 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpgkr5r73g/model/ with prefix 4d4ff72448154b6c
[INFO 24-10-30 03:29:59.5551 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:59.6699 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:59.6700 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:59.6700 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:22


[INFO 24-10-30 03:29:59.8560 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmph9apj7j0/model/ with prefix d2ba853dc3024d02
[INFO 24-10-30 03:29:59.8587 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:29:59.9727 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:59.9727 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:29:59.9727 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:23


[INFO 24-10-30 03:30:00.1372 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpvgxzwyfo/model/ with prefix d4e0e6d0d380497f
[INFO 24-10-30 03:30:00.1391 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:00.2553 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:00.2553 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:00.2553 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:24


[INFO 24-10-30 03:30:00.4194 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp25z71tof/model/ with prefix 94899267516a4a2b
[INFO 24-10-30 03:30:00.4212 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:00.5368 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:00.5368 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:00.5368 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:25


[INFO 24-10-30 03:30:00.7156 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpc49fpe52/model/ with prefix a43b2c38791b4a09
[INFO 24-10-30 03:30:00.7182 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:00.8338 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:00.8338 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:00.8338 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:26


[INFO 24-10-30 03:30:00.9896 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp8au2u_m6/model/ with prefix 7c62ca9bfc694ec5
[INFO 24-10-30 03:30:00.9912 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:01.1049 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:01.1049 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:01.1049 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:27


[INFO 24-10-30 03:30:01.3103 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpo6ap_a4g/model/ with prefix d87c810575424faf
[INFO 24-10-30 03:30:01.3139 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:01.4282 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:01.4282 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:01.4282 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:28


[INFO 24-10-30 03:30:01.5736 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpearsd7mg/model/ with prefix fc39d0febdf44db5
[INFO 24-10-30 03:30:01.5747 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:01.7071 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:01.7071 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:01.7071 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:29


[INFO 24-10-30 03:30:01.8645 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpbjmzingf/model/ with prefix 22e5b740d03d4f3a
[INFO 24-10-30 03:30:01.8661 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:01.9802 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:01.9802 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:01.9802 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:30


[INFO 24-10-30 03:30:02.1783 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmph4wuqhxs/model/ with prefix d6fa1759ce0a405a
[INFO 24-10-30 03:30:02.1815 +07 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-10-30 03:30:02.1815 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:02.2970 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:02.2970 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:02.2970 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:31


[INFO 24-10-30 03:30:02.5384 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp1fxijrin/model/ with prefix 7d39da686afa4853
[INFO 24-10-30 03:30:02.5430 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:02.6566 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:02.6566 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:02.6567 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:32


[INFO 24-10-30 03:30:02.9581 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpcpktp3n5/model/ with prefix d0ad667adffe4aed
[INFO 24-10-30 03:30:02.9603 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:03.0810 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:03.0810 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:03.0810 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:33


[INFO 24-10-30 03:30:03.2666 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp_75517n3/model/ with prefix 1a30229befd44015
[INFO 24-10-30 03:30:03.2692 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:03.3873 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:03.3873 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:03.3873 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:34


[INFO 24-10-30 03:30:03.5561 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmprxxmk2q8/model/ with prefix b7b553da773d4cf7
[INFO 24-10-30 03:30:03.5580 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:03.6752 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:03.6752 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:03.6752 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:35


[INFO 24-10-30 03:30:03.8639 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpoqa4wq77/model/ with prefix bef92dd82cc04bc3
[INFO 24-10-30 03:30:03.8666 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:03.9831 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:03.9832 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:03.9832 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:36


[INFO 24-10-30 03:30:04.1477 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp18pedo5h/model/ with prefix 88b27efc3d214126
[INFO 24-10-30 03:30:04.1494 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:04.2656 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:04.2656 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:04.2656 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:37


[INFO 24-10-30 03:30:04.4411 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpnlccr33e/model/ with prefix f7e2c229c02c4c94
[INFO 24-10-30 03:30:04.4428 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:04.5601 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:04.5601 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:04.5602 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:38


[INFO 24-10-30 03:30:04.7514 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmplrn4tsmo/model/ with prefix 6e840b1b48cd4138
[INFO 24-10-30 03:30:04.7543 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:04.8903 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:04.8903 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:04.8903 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:39


[INFO 24-10-30 03:30:05.0961 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpdbn0le67/model/ with prefix ed05382abaef40e7
[INFO 24-10-30 03:30:05.0995 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:05.2171 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:05.2172 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:05.2172 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:40


[INFO 24-10-30 03:30:05.3881 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp85tu4p20/model/ with prefix b76be6c742884fb9
[INFO 24-10-30 03:30:05.3901 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:05.5265 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:05.5266 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:05.5266 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:41


[INFO 24-10-30 03:30:05.7282 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmplir8d28m/model/ with prefix 7d4f347f59d9427f
[INFO 24-10-30 03:30:05.7313 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:05.8468 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:05.8468 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:05.8468 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:42


[INFO 24-10-30 03:30:06.0213 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpg8968xj9/model/ with prefix 1d4eeb25e5ba40f8
[INFO 24-10-30 03:30:06.0235 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:06.1389 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:06.1389 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:06.1389 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:43


[INFO 24-10-30 03:30:06.3306 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpfxemsx86/model/ with prefix b995ef3de8a44732
[INFO 24-10-30 03:30:06.3334 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:06.4503 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:06.4503 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:06.4503 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:44


[INFO 24-10-30 03:30:06.6228 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpdwo87vqv/model/ with prefix acbe5aeeed8b401a
[INFO 24-10-30 03:30:06.6248 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:06.7653 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:06.7653 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:06.7653 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:45


[INFO 24-10-30 03:30:06.9156 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpytrdp451/model/ with prefix c880a272026f460e
[INFO 24-10-30 03:30:06.9167 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:07.0340 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:07.0341 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:07.0341 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:46


[INFO 24-10-30 03:30:07.2218 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpqygfx1jb/model/ with prefix aadcfd913f504b7e
[INFO 24-10-30 03:30:07.2245 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:07.3394 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:07.3394 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:07.3394 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:47


[INFO 24-10-30 03:30:07.5095 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpakm7udua/model/ with prefix 623835814cea459b
[INFO 24-10-30 03:30:07.5115 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:07.6305 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:07.6305 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:07.6305 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:48


[INFO 24-10-30 03:30:07.7917 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp6mco__gn/model/ with prefix aa3756658d1a4441
[INFO 24-10-30 03:30:07.7932 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:07.9085 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:07.9085 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:07.9085 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:49


[INFO 24-10-30 03:30:08.0894 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp9lb7lr4v/model/ with prefix 35732996eea8403c
[INFO 24-10-30 03:30:08.0919 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:08.2082 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:08.2082 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:08.2082 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:50


[INFO 24-10-30 03:30:08.4247 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpp46bbw54/model/ with prefix c8babed9fb18478e
[INFO 24-10-30 03:30:08.4285 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:08.5434 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:08.5434 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:08.5434 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:51


[INFO 24-10-30 03:30:08.7728 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpbvwettkd/model/ with prefix 38b6a656b60c4d41
[INFO 24-10-30 03:30:08.7770 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:08.8909 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:08.8909 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:08.8909 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:52


[INFO 24-10-30 03:30:09.0666 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpy8xzqtmb/model/ with prefix 7ed449f5524c4405
[INFO 24-10-30 03:30:09.0690 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:09.2056 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:09.2056 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:09.2056 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:53


[INFO 24-10-30 03:30:09.5157 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp5zhu5gtj/model/ with prefix 9fcd00cb9ca64414
[INFO 24-10-30 03:30:09.5180 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:09.6391 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:09.6391 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:09.6391 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:54


[INFO 24-10-30 03:30:09.8257 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpjowhoa9i/model/ with prefix 0b9a168ab0f74908
[INFO 24-10-30 03:30:09.8280 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:09.9530 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:09.9530 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:09.9530 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:55


[INFO 24-10-30 03:30:10.1564 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp1494lno4/model/ with prefix 76c76233a5954a0f
[INFO 24-10-30 03:30:10.1594 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:10.2791 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:10.2792 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:10.2792 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:56


[INFO 24-10-30 03:30:10.4677 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpfv0r4mn1/model/ with prefix 44c5ae6f2c1b4c64
[INFO 24-10-30 03:30:10.4702 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:10.5962 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:10.5963 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:10.5963 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:57


[INFO 24-10-30 03:30:10.7607 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpww0g2diu/model/ with prefix 9bf710ed08d94563
[INFO 24-10-30 03:30:10.7620 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:10.8861 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:10.8861 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:10.8861 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:58


[INFO 24-10-30 03:30:11.1457 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpzwpj9yqc/model/ with prefix daa978a9043c4239
[INFO 24-10-30 03:30:11.1505 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:11.2733 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:11.2733 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:11.2733 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:59


[INFO 24-10-30 03:30:11.5215 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpai7pa5da/model/ with prefix 368c7edf02a54ac5
[INFO 24-10-30 03:30:11.5258 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:11.6500 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:11.6500 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:11.6501 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:60


[INFO 24-10-30 03:30:11.8325 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp0jn3l801/model/ with prefix bcfe88981ffc4489
[INFO 24-10-30 03:30:11.8339 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:11.9504 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:11.9504 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:11.9504 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:61


[INFO 24-10-30 03:30:12.1290 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpdimzqv3t/model/ with prefix 8114f0c10b80448c
[INFO 24-10-30 03:30:12.1312 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:12.2498 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:12.2498 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:12.2498 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:62


[INFO 24-10-30 03:30:12.4452 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp0cu1gqj4/model/ with prefix 329c8747a6a74852
[INFO 24-10-30 03:30:12.4479 +07 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-10-30 03:30:12.4479 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:12.5707 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:12.5707 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:12.5707 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:63


[INFO 24-10-30 03:30:12.7293 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp1pk60qhg/model/ with prefix 2e57163f394246b1
[INFO 24-10-30 03:30:12.7307 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:12.8506 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:12.8506 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:12.8506 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:64


[INFO 24-10-30 03:30:13.0348 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp_cyqyk86/model/ with prefix b359aae15e684ffb
[INFO 24-10-30 03:30:13.0375 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:13.1601 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:13.1601 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:13.1601 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:65


[INFO 24-10-30 03:30:13.3670 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpkggd9in_/model/ with prefix 75e7bfa8c8f44325
[INFO 24-10-30 03:30:13.3703 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:13.4913 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:13.4913 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:13.4913 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:66


[INFO 24-10-30 03:30:13.6642 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpyt_3joti/model/ with prefix 100e57b776ad4d23
[INFO 24-10-30 03:30:13.6658 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:13.7881 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:13.7881 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:13.7881 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:67


[INFO 24-10-30 03:30:13.9873 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpmi03dcj6/model/ with prefix d22d52a8763949f2
[INFO 24-10-30 03:30:13.9899 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:14.1401 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:14.1401 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:14.1401 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:68


[INFO 24-10-30 03:30:14.3616 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpg2reawyq/model/ with prefix dd56d4294fd04eee
[INFO 24-10-30 03:30:14.3649 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:14.4814 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:14.4815 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:14.4815 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:69


[INFO 24-10-30 03:30:14.6913 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp89pn_jw7/model/ with prefix 6ed695670d354a81
[INFO 24-10-30 03:30:14.6961 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:14.8230 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:14.8231 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:14.8231 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:70


[INFO 24-10-30 03:30:14.9986 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpicpiifrb/model/ with prefix aa00a753a1154e07
[INFO 24-10-30 03:30:15.0004 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:15.1173 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:15.1173 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:15.1174 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:71


[INFO 24-10-30 03:30:15.2761 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp01s073s9/model/ with prefix 0e21cb3e238944ed
[INFO 24-10-30 03:30:15.2775 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:15.4002 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:15.4002 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:15.4003 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:72


[INFO 24-10-30 03:30:15.5950 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpbkit7dxb/model/ with prefix 5061bd5340b6498d
[INFO 24-10-30 03:30:15.5974 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:15.7152 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:15.7152 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:15.7152 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:73


[INFO 24-10-30 03:30:15.8908 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp646gd2sh/model/ with prefix 1f75b27e77e84316
[INFO 24-10-30 03:30:15.8920 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:16.0074 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:16.0075 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:16.0075 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:74


[INFO 24-10-30 03:30:16.2538 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpghr0691m/model/ with prefix 9838bb8674f742d2
[INFO 24-10-30 03:30:16.2582 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:16.3719 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:16.3719 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:16.3719 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:75


[INFO 24-10-30 03:30:16.5344 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpo2941ldr/model/ with prefix 8a87e1a7b82d42b8
[INFO 24-10-30 03:30:16.5361 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:16.6542 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:16.6543 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:16.6543 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:76


[INFO 24-10-30 03:30:16.8210 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpa14r8z9j/model/ with prefix 82ac270058974c00
[INFO 24-10-30 03:30:16.8228 +07 kernel.cc:1061] Use fast generic engine


i:77


[WARNING 24-10-30 03:30:17.0988 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:17.0988 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:17.0988 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-10-30 03:30:17.3208 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpwfubvjxx/model/ with prefix 6f266a834f254118
[INFO 24-10-30 03:30:17.3236 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:17.4519 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:17.4519 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:17.4519 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:78


[INFO 24-10-30 03:30:17.6042 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpw_wrxtuc/model/ with prefix 7ceb5edf5e6c4ac3
[INFO 24-10-30 03:30:17.6054 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:17.7293 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:17.7293 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:17.7293 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:79


[INFO 24-10-30 03:30:17.9392 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpio3s_h89/model/ with prefix 7cfc82bfc383476a
[INFO 24-10-30 03:30:17.9435 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:18.0638 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:18.0638 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:18.0638 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:80


[INFO 24-10-30 03:30:18.2386 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpdz1n41wp/model/ with prefix e74306074c084080
[INFO 24-10-30 03:30:18.2405 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:18.3582 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:18.3583 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:18.3583 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:81


[INFO 24-10-30 03:30:18.5549 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp9d49s1v2/model/ with prefix f1709fbcd6164cac
[INFO 24-10-30 03:30:18.5577 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:18.6762 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:18.6763 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:18.6763 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:82


[INFO 24-10-30 03:30:18.8803 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpx8pla_nx/model/ with prefix 2e1341e8b2bb4131
[INFO 24-10-30 03:30:18.8835 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:19.0012 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:19.0012 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:19.0012 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:83


[INFO 24-10-30 03:30:19.2062 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp4ucshuns/model/ with prefix 22913bb95c9e4bdc
[INFO 24-10-30 03:30:19.2093 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:19.3278 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:19.3278 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:19.3278 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:84


[INFO 24-10-30 03:30:19.5478 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpi32i_3mf/model/ with prefix b78703e06bda4b41
[INFO 24-10-30 03:30:19.5507 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:19.6688 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:19.6688 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:19.6689 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:85


[INFO 24-10-30 03:30:19.8586 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp3s0w1rg8/model/ with prefix 35227112d7964e08
[INFO 24-10-30 03:30:19.8611 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:19.9854 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:19.9854 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:19.9855 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:86


[INFO 24-10-30 03:30:20.2408 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmps1m6c8q4/model/ with prefix 9a103833af064d60
[INFO 24-10-30 03:30:20.2451 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:20.3665 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:20.3665 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:20.3665 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:87


[INFO 24-10-30 03:30:20.5534 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpfjce8re7/model/ with prefix 28f3729e341647b4
[INFO 24-10-30 03:30:20.5556 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:20.6820 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:20.6820 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:20.6820 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:88


[INFO 24-10-30 03:30:20.8361 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpp37ipq44/model/ with prefix 5030e10a825b41ee
[INFO 24-10-30 03:30:20.8374 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:20.9574 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:20.9574 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:20.9574 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:89


[INFO 24-10-30 03:30:21.1426 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpnc3rq4p1/model/ with prefix 9823538af5f34385
[INFO 24-10-30 03:30:21.1449 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:21.2630 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:21.2631 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:21.2631 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:90


[INFO 24-10-30 03:30:21.4426 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpq96lw567/model/ with prefix 81a94db754594156
[INFO 24-10-30 03:30:21.4447 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:21.5799 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:21.5799 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:21.5800 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:91


[INFO 24-10-30 03:30:21.7894 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpacooj026/model/ with prefix 2945ed7b954d4ebb
[INFO 24-10-30 03:30:21.7936 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:21.9079 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:21.9079 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:21.9080 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:92


[INFO 24-10-30 03:30:22.1939 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpolsuinyd/model/ with prefix f1028b0847f740c8
[INFO 24-10-30 03:30:22.1996 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:22.3143 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:22.3144 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:22.3144 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:93


[INFO 24-10-30 03:30:22.4801 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp3dh43232/model/ with prefix 3f59921478d749fc
[INFO 24-10-30 03:30:22.4816 +07 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-10-30 03:30:22.4817 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:22.6036 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:22.6036 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:22.6036 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:94


[INFO 24-10-30 03:30:22.7816 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpuei0w4ba/model/ with prefix f4ff70e003b44044
[INFO 24-10-30 03:30:22.7836 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:22.9037 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:22.9037 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:22.9037 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:95


[INFO 24-10-30 03:30:23.0861 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmplm30cvm5/model/ with prefix eeaab4724c73413b
[INFO 24-10-30 03:30:23.0883 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:23.2067 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:23.2067 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:23.2067 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:96


[INFO 24-10-30 03:30:23.3969 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp6dya904c/model/ with prefix 70d237ebd4b7408f
[INFO 24-10-30 03:30:23.3992 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:23.5176 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:23.5176 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:23.5176 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:97


[INFO 24-10-30 03:30:23.7196 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmpj82hhv9p/model/ with prefix 01a64704eba14cd4
[INFO 24-10-30 03:30:23.7211 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:23.8412 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:23.8412 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:23.8412 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:98


[INFO 24-10-30 03:30:24.1682 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp7_42puj4/model/ with prefix a7d31f7d51b3453b
[INFO 24-10-30 03:30:24.1747 +07 kernel.cc:1061] Use fast generic engine
[WARNING 24-10-30 03:30:24.2900 +07 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:24.2901 +07 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-10-30 03:30:24.2901 +07 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:99
Submission exported to submission.csv


[INFO 24-10-30 03:30:24.6535 +07 kernel.cc:1233] Loading model from path /var/folders/3f/66_mwvt95b99cr0msjcyfm040000gn/T/tmp5o29cx08/model/ with prefix 18ee105c7c5a4333
[INFO 24-10-30 03:30:24.6619 +07 kernel.cc:1061] Use fast generic engine


## Algorithm Learned so far
### Tensorflow 
- used for deep learning and neural network models, especially suited for unstructured data like images, text, and audio. It supports complex model building, automatic differentiation, and is deployable across different platforms.

### Tensorflow Decision Forest
- specialized library within TensorFlow for tree-based models, including Random Forests and Gradient Boosted Trees. These models work well with structured/tabular data, require minimal preprocessing, and are generally easier to interpret.

### Ensemble
- technique where  multiple models are combined to solve the same problem. Rather than relying on a single model, ensemble methods merge the predictions of several models to improve performance, accuracy, and robustness.
- They often yield more accurate predictions by averaging out errors and reducing model variance
- Generally less prone to overfitting,

